### ISSUE TITLE
where to put exercise answers ?

### ISSUE THREAD

**Issue by andysouth** [afrimapr/afrimapr-book#16]
Discussing exercise answers with @martynabogacz, also demonstrating using github issues for sharing book decision making.

TLDR summary: lets put exercise answers at the end of each chapter for now & can change later if needed :-)

In the previous book meeting we decided that we would have 2 types of exercise. Small exercise to go at the end of each episode, and bigger 'putting into practice' sections to go at the end of chapters.

We also talked about where to put exercise solutions. @Julie-AnneTangena said good to have solutions next to exercises. @martynabogacz says good to have separated by something so that learners don't just copy without thinking. @LaurieLBaker asked whether we could use an interactive tab solution like in the learnr tutorials (but we think this may be tricky in the book). In the geocompr book @Robinlovelace puts the solutions in a separate website https://geocompr.github.io/vignettes/ which has the advantage that they can be edited separate to the book but slight disadvantage that they are more difficult to find.

For now we will go with having the exercise solutions at the end of each chapter and then we can always move them later either closer to the questions or to the end of the book. Similarly we can put the 'putting into practice' solutions immediately after their section and can move later.

Comments welcome.

**Comment by Robinlovelace** [comment]
Good question. I suggest placing small ones before the end of each chapter to encourage it to be a working document. Tip I've just thought of: treat exercises as fully fledged sub-sections. Note the sections here are good but you cannot link to them: https://itsleeds.github.io/rrsrr/basics.html#object-types-vectors-and-data-frames
